package.json

```diff
@@ -2,96 +2,62 @@
   "name": "tga-spectrum",
   "version": "0.17.0",
   "description": "tga-spectrum provides tools to convert output files from thermogravimetric analysis (TGA) into JCAMP-DX, as well as tools to analyze the data (mass loss analysis)",
-  "main": "lib/index.js",
-  "module": "src/index.js",
+  "main": "./lib/index.js",
+  "module": "./lib-esm/index.js",
+  "types": "./lib/index.d.ts",
+  "keywords": [],
+  "author": "Luc Patiny",
+  "license": "MIT",
   "files": [
+    "src",
     "lib",
-    "src"
+    "lib-esm"
   ],
   "scripts": {
-    "build": "cheminfo-build  --entry src/index.js --root TGASpectrum",
-    "build-doc": "cheminfo doc",
-    "eslint": "eslint src",
+    "build": "npm run tsc-cjs && cheminfo-build  --entry lib/index.js --root TGASpectrum",
+    "check-types": "tsc --noEmit",
+    "clean": "rimraf lib lib-esm",
+    "eslint": "eslint src --cache",
     "eslint-fix": "npm run eslint -- --fix",
-    "prepublishOnly": "rollup -c",
-    "test": "npm run test-coverage && npm run eslint",
-    "test-coverage": "jest --coverage",
-    "test-only": "jest"
+    "prepack": "npm run tsc",
+    "prettier": "prettier --check src",
+    "prettier-write": "prettier --write src",
+    "test": "npm run test-coverage && npm run eslint && npm run prettier && npm run check-types",
+    "test-coverage": "npm run test-only -- --coverage",
+    "test-only": "jest",
+    "tsc": "npm run clean && npm run tsc-cjs && npm run tsc-esm",
+    "tsc-cjs": "tsc --project tsconfig.cjs.json",
+    "tsc-esm": "tsc --project tsconfig.esm.json"
   },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/cheminfo/tga-spectrum.git"
   },
-  "keywords": [],
-  "author": "Luc Patiny",
-  "license": "MIT",
   "bugs": {
     "url": "https://github.com/cheminfo/tga-spectrum/issues"
   },
   "homepage": "https://github.com/cheminfo/tga-spectrum#readme",
   "jest": {
+    "preset": "ts-jest",
     "testEnvironment": "node"
   },
-  "devDependencies": {
-    "@babel/plugin-transform-modules-commonjs": "^7.15.4",
-    "@types/jest": "^27.0.1",
-    "cheminfo-build": "1.1.11",
-    "codecov": "^3.8.3",
-    "eslint": "^7.32.0",
-    "eslint-config-cheminfo": "^5.5.0",
-    "eslint-plugin-import": "^2.24.2",
-    "eslint-plugin-jest": "^24.4.0",
-    "eslint-plugin-prettier": "^4.0.0",
-    "esm": "^3.2.25",
-    "jest": "^27.2.0",
-    "prettier": "^2.4.0",
-    "rollup": "^2.56.3"
-  },
   "dependencies": {
-    "common-spectrum": "0.40.0",
-    "ensure-string": "^1.0.0",
+    "@types/react": "^17.0.24",
+    "common-spectrum": "0.43.1",
+    "ensure-string": "^1.1.0",
     "papaparse": "^5.3.1",
-    "xlsx": "^0.17.1"
+    "xlsx": "^0.17.2"
   },
-  "info": {
-    "logo": "https://raw.githubusercontent.com/cheminfo/font/master/src/tga/assignment.svg",
-    "domain": [
-      "Physical Chemistry",
-      "Materials Science"
-    ],
```

Comment:
what was this?

Reply:
I have put it back. This is some info about the package in order to generate automatically the documentation about all the converters we have. Kevin is working on this.